cbatticon.c

```diff
@@ -696,6 +696,16 @@ static void reset_battery_time_estimation (void)
  * tray icon functions
  */
 
+static void set_icon(GtkStatusIcon *tray_icon, const gchar* name) {
+    GdkPixbuf *pix = gtk_icon_theme_load_icon(gtk_icon_theme_get_default(),
+                                              name,
+                                              32,
```

Comment:
@johannesk, will the fixed 32 value not be a problem for the person not using icon size of 32?

Reply:
Thats correct. I pushed another commit, which gets the actual icon size.